lms/djangoapps/courseware/views/index.py

```diff
@@ -437,6 +437,13 @@ def _create_courseware_context(self):
                 table_of_contents['next_of_active_section'],
             )
             courseware_context['fragment'] = self.section.render(STUDENT_VIEW, section_context)
+            if self.section.position and self.section.has_children:
+                display_items = self.section.get_display_items()
+                if len(display_items) > 0:
+                    courseware_context['section_title'] = '{position_title} | {section_title}'.format(
+                        position_title=display_items[self.section.position - 1].display_name_with_default,
+                        section_title=courseware_context['section_title']
+                    )
```

Comment:
Please check for an out of bounds error here. In edge cases, it is possible for a user to have a saved index value for something that no longer exists -- for example, if someone put out a learning sequence with 10 units, but then moved a couple of those units to the next sequence. In general, we just want to be really forgiving here, and default to the first unit if it's out of bounds or even nonsensical (e.g. -1).